Scrapper-Service/database/mdb.py

```diff
@@ -56,15 +57,15 @@ def __del__(self):
         self.logger.info("Closing connection to mongodb !!")
         self.client.close()
         self.logger.info("Succesfully Closed connection to mongodb !!")
-
+            
     def put(self , conference_data):
         if not isinstance(conference_data , Conference):
             raise ValueError("Provided data is not in proper format as required by db")
         else:
             _id = conference_data._id
             try:
-                res = self.collection.update_one( {'_id':_id}  ,conference_data.get_query(), upsert = True)
+                res = self.collection.update_one( {'_id':_id,'$or':[ {'deadline':{'$gte':conference_data.querydata['deadline']}}, {'dateRange':{'$ne':conference_data.querydata['dateRange']}}] }  ,conference_data.get_query(), upsert = True)
```

Comment:
* Can you check this once whether it should be lesser than or greater than ?
* we want condition to be True when "new incoming data" date is greater than what is contained in db. Check how the gte is expecting its argument.

Reply:
it would check existing data is greater than incoming data